backend/nexusInteraction.py

```diff
@@ -93,7 +93,7 @@ def runData(file, fields, run):
 
 
 def runFields(instrument, cycle, runs):
-    runArr = runs.split(";")
+    runArr = runs.split("-")
```

Comment:
I can't get the nexus fields when I have more than one run selected - here you changed the delimited from `;` to `-`, but the error message seems to suggest that you're still adding `;` to the URL:
```
Error2: Error transferring http://127.0.0.1:5000/getNexusFields/nimrod/cycle_21_1/72544;72545;72546 - server replied: INTERNAL SERVER ERROR
```

Reply:
Oops! I know what I've done there